.vuepress/config.js

```diff
@@ -29,7 +29,14 @@ module.exports = {
                     'prototyping-tools/',
 
                 ]
-            }]
+            }],
+            '/resources/REST/': [{
```

Comment:
I don't think we need to organize the REST page as a nested structure. Larger topics like Design or web development require it since they have a wealth of content.